### ISSUE TITLE
cwe_checker: Import failed for file: /input (HeadlessAnalyzer)  

### ISSUE THREAD

**Issue by KongLynn** [fkie-cad/cwe_checker#229]
upload firmware and scan, there are many errors in the terminal like this:
[cwe_checker][ERROR]: cwe_checker execution failed: Execution of Ghidra plugin failed: Process was terminated.
ERROR REPORT: Import failed for file: /input (HeadlessAnalyzer)

**Comment by Enkelmann** [comment]
I assume you are running the cwe_checker through FACT? Can you provide an example binary from your firmware where this happens?

I will also look into improving the error message in FACT so that it includes a proper file path instead of the useless `/input` artifact.

**Comment by KongLynn** [comment]
I don't know how to get the example binary ,the firmware I used for rail can be download at https://www.tp-link.com/us/support/download/archer-a7/#Firmware

**Comment by Enkelmann** [comment]
Thanks! That should be enough for me to find an example binary for the error message. I am on it.

**Comment by Enkelmann** [comment]
Ok, I managed to track down an example of this. The problem is that Ghidra, which we use to disassemble a binary, throws an error and fails to disassemble the file. This is either due to a bug in Ghidra or due to the binary itself being broken. The latter case is more likely here, since the unpacking process of a firmware can generate a lot of incorrectly unpacked files. Which also would explain why there are many of these error messages in the terminal log. The example I tracked down was also a broken ELF file because of an incorrectly unpacked file.

So there is nothing we can do on the cwe_checker side, since Ghidra claims that the file is broken (which is very likely correct, i.e. not even a bug in Ghidra) and the cwe_checker cannot do anything if Ghidra does not want to disassemble a file.

Long story short: As long as the error is not generated for a binary where you think that the binary itself is correctly unpacked just ignore these error messages in FACT.
